abnf/distributive.txt

```diff
@@ -0,0 +1,15 @@
+ALPHA = %x41-5A / %x61-7A
+COMMA = ","
+
+odataIdentifier= ALPHA *ALPHA
+entitySetName = odataIdentifier
+keyPredicate = "('DEFAULT')"
+entityColNavigationProperty = odataIdentifier
+
+recHierReference = ( entitySetName keyPredicate "/" entityColNavigationProperty
+                   / entitySetName 
```

Comment:
You have to put the longer match first because the parser takes the first matching alternative